pontoon/base/static/js/translate.js

```diff
@@ -1916,7 +1938,8 @@ var Pontoon = (function (my) {
       this.user = {
         email: $('#server').data('email') || '',
         name: $('#server').data('name') || '',
-        localizer: isLocalizer($('#server').data('user-translated-locales')),
+        forceSuggestions: $('.force-suggestions').is('.enabled'),
```

Comment:
It seems weird to read this from the state of the checkbox vs the server data tag. The box being checked isn't actually strongly linked to what the real value on the server is (because maybe autocomplete screws up keeping it checked or the user's JS loads slowly and they click the checkbox before we reach this point).